docker/build_docker_images.sh

```diff
@@ -11,5 +11,9 @@ docker build -f ${BASEDIR}/Dockerfile-rhel-8 -t vpl:rhel8 ${BASEDIR}/..
 docker build -f ${BASEDIR}/Dockerfile-centos-7 -t vpl:centos7 ${BASEDIR}/..
 docker build -f ${BASEDIR}/Dockerfile-centos-8 -t vpl:centos8 ${BASEDIR}/..
 docker build -f ${BASEDIR}/Dockerfile-ubuntu-18.04 -t vpl:ubuntu18.04 ${BASEDIR}/..
-docker build -f ${BASEDIR}/Dockerfile-ubuntu-19.10 -t vpl:ubuntu19.10 ${BASEDIR}/..
+
+if [ "$(uname -m)" == "x86_64" ]; then
```

Comment:
Does ubuntu 19.10 not support ARM?

Reply:
Correct, ubuntu 19.10 is EOL and binaries not available at http://ports.ubuntu.com/dists/
I'm curious why Intel is trying support an EOL disto for any architecture?